verify-sna/functions/check-verify.js

```diff
@@ -1,79 +1,243 @@
 /**
- * Check Verification
+ * Check a verification
```

Comment:
Before I forget, remember that we agreed with the demo app side that the endpoints will be called verify-start and verify-check correspondingly.

Reply:
Changed.